gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,10 +1,6 @@
-#Tue May 01 11:34:35 PDT 2018
-
+#Fri Jun 01 19:51:59 EST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-android.enableD8.desugaring=true
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-bin.zip
-# Android Studio Gradle wrapper 3.1.2 doesn't support configure on demand
-org.gradle.configureondemand=false
\ No newline at end of file
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
```

Comment:
Please revert all changes to this file. 😊

Reply:
oh yes! this always slips in.